### ISSUE TITLE
Umezawa's Jitte didn't get counters when creature dealt damage but was immediately exiled.

### ISSUE THREAD

**Issue by Grath** [magefree/mage#8555]
9:19 PM, T21.M1: MURPH activates: Equip. from Umezawa's Jitte [fb7] targeting Warrior [4e2]
9:19 PM, T21.DA: MURPH attacks with 1 creature
9:20 PM, T21.DB: Attacked player: Grath
9:20 PM, T21.DB: Attacker: Warrior [4e2] (1/1) blocked by Gisa, Glorious Resurrector [7c1] (4/4) 
9:20 PM, T21.CD: Warrior [4e2] deals 1 damage to Gisa, Glorious Resurrector [7c1]
9:20 PM, T21.CD: Gisa, Glorious Resurrector [7c1] deals 4 damage to Warrior [4e2]
9:20 PM, T21.CD: Grath moves Warrior [4e2] from battlefield to the exile zone (source: Gisa, Glorious Resurrector [7c1])

Umezawa's Jitte didn't get any counters, the ability didn't even trigger.

**Comment by stevedavidmusic** [comment]
It doesn't trigger if the equipped creature doesn't survive combat.

**Comment by stevedavidmusic** [comment]
Here's a screenshot of two Llanwar Elves trading with each other in combat. Both of them died in combat and neither of their Jittes triggered to gain counters. 

<img width="1280" alt="Screen Shot 2022-02-16 at 8 22 50 PM" src="https://user-images.githubusercontent.com/45299811/154392576-ad22146d-7db9-4874-8190-69b18bfe3a08.png">

<img width="1280" alt="Screen Shot 2022-02-16 at 8 23 20 PM" src="https://user-images.githubusercontent.com/45299811/154392585-dc63fc50-891a-41da-adf4-7325b8365b48.png">

**Comment by Alex-Vasile** [comment]
Duplicate of #9129. See that one for more in-depth discussion about fixing it.
